dictionary.json

```diff
@@ -18,6 +18,7 @@
     "keywords": [],
     "frame": "c",
     "distribution": "d",
+    "animacy": "maq",
```

Comment:
I think it would be nice to use the term 'pronomial class' rather than 'animacy' for consistency with the refgram, though it's ultimately not important. Also, the refgram makes it look like the primary way to refer to the classes is by their number rather than the associated pronouns.